README.md

```diff
@@ -257,26 +257,10 @@ Kyo provides direct syntax for a more intuitive and concise way to express compu
 
 Essentially, `await` is a syntactic sugar for the `map` function, allowing developers to directly access values from computations without the need for repetitive `map` chaining. This makes the code more linear and intuitive.
 
-```scala
-import kyo._
+```scala mdoc:passthrough
+import kyo.readmeExamples.utils._
 
-// Use the direct syntax
-val a: String < (Aborts[Exception] & Options) =
-  defer {
-    val b: String = 
-      await(Options.get(Some("hello")))
-    val c: String = 
-      await(Aborts.get(Right("world")))
-    b + " " + c
-  }
-
-// Equivalent desugared
-val b: String < (Aborts[Exception] & Options) =
-  Options.get(Some("hello")).map { b =>
-    Aborts.get(Right("world")).map { c =>
-      b + " " + c
-    }
-  }
+printSource("readme-examples/shared/src/main/scala/kyo/Example1.scala")
```

Comment:
Hi @987Nabil I went with the approach used in zio-http but I noticed that this function only works when I use the absolute path to the file containing the example for some reason. Also I noticed that the output compiles in `target\README-out.md`. Should I copy this file to the main directory as the new README ?

Reply:
I am not the expert on the doc generation of zio-http, but seems reasonable :)